tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -105,6 +199,70 @@ static void test_advx_main(void)
 
 	k_sleep(K_MSEC(1000));
 
+	printk("Create advertising set...");
+	err = bt_le_ext_adv_create(BT_LE_ADV_CONN_NAME, &adv_callbacks, &adv);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	printk("Start advertising using extended commands (max_events)...");
+	is_sent = false;
+	ext_adv_param.timeout = 0;
+	ext_adv_param.num_events = 3;
+	err = bt_le_ext_adv_start(adv, &ext_adv_param);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	printk("Waiting...");
+	while (is_sent) {
+		k_sleep(K_MSEC(100));
+	}
+	printk("done.\n");
```

Comment:
Even though we say that we are done, we are not testing that advertisement has stopped after the correct number of events. 
in other words: when we come here it can be for several reasons that should cause the test to fail
- advertisement never started
- advertisement has started, but is not stopped
- advertisement has started, and is stopped, but after the wrong number of events

Reply:
Check for num_sent added on in the advx test. Test to check for adv reports in scanx test is todo, postponed to be added as part of periodic advertising testing.

Reply:
Ok, looking forward to this in the next PR